README.md

```diff
@@ -8,10 +8,53 @@ razeedeploy. It retrieves and applies the configuration for all resources.
 
 ## Install
 
-```shell
-kubectl apply -f "https://github.com/razee-io/RemoteResource/releases/latest/download/resource.yaml"
+[Razee Deploy Delta](https://github.com/razee-io/razeedeploy-delta) is the
+recommended way to install RemoteResource.
+
+## Customize the Controller
```

Comment:
im not sure how i feel about this being at the top of the page. i think its important the suer knows not to enable user impersonation without webhook installed, but i dont think this needs to be before they even know what a RemoteResource example looks like. i think it makes more sense closer to the bottom of the page